File: fixtures/scale/javascript/small/border.js
import theme from './theme.js';

function borderTop(width, name) {
  const chars = theme.themeChars(name);
  return chars.corner + chars.edge.repeat(width) + chars.corner;
}

function borderSide(line, width, name) {
  const chars = theme.themeChars(name);
  return chars.side + line.padEnd(width) + chars.side;
}

export default { borderTop, borderSide };

File: fixtures/scale/python/small/border.py
from theme import theme_chars


def border_top(width, theme):
    chars = theme_chars(theme)
    return chars["corner"] + chars["edge"] * width + chars["corner"]


def border_side(line, width, theme):
    chars = theme_chars(theme)
    return chars["side"] + line.ljust(width) + chars["side"]
